experiments/AtmosGCM/heldsuarez.jl

```diff
@@ -151,13 +151,13 @@ function held_suarez_forcing!(
 end
 
 function config_diagnostics(FT, driver_config)
-    interval = "100000steps" # chosen to allow a single diagnostics collection
+    interval = "40000steps" # chosen to allow a single diagnostics collection
 
-    _planet_radius = FT(planet_radius(param_set))
+    _planet_radius = planet_radius(param_set)
```

Comment:
In case this variable is used again, wouldn’t it be safer to type-cast once here?